base/applications/taskmgr/procpage.c

```diff
@@ -931,3 +933,160 @@ int CALLBACK ProcessPageCompareFunc(LPARAM lParam1, LPARAM lParam2, LPARAM lPara
     }
     return ret;
 }
+
+/* FIXME: Use this code to get process executable path if CORE-16145 was fixed. */
+#if 0
+static BOOL GetProcessExecutablePath(DWORD dwProcessId, LPWSTR lpExePath, DWORD dwLength)
+{
+    typedef BOOL (WINAPI *PFN_QUERYFULLPROCESSIMAGENAMEW)(HANDLE, DWORD, LPWSTR, PDWORD);
+
+    HANDLE hProcess;
+    HMODULE hLibrary;
+    PFN_QUERYFULLPROCESSIMAGENAMEW _QueryFullProcessImageNameW;
+    DWORD dwPathLen;
+    BOOL bRet = FALSE;
+
+    if (dwProcessId == 0)
+    {
+        return FALSE;
+    }
+
+    hProcess = OpenProcess(PROCESS_QUERY_INFORMATION, FALSE, dwProcessId);
+
+    if (!hProcess)
+    {
+        return FALSE;
+    }
+
+    hLibrary = LoadLibraryW(L"kernel32_vista.dll");
```

Comment:
I don't think the taskmgr ever should use anything from kernel32_vista. Neither today, nor in the future.
Other applications, like Mozilla Firefox can absolutely offer similar functionality, without using that.
What is the benefit of this GetProcessExecutablePath() implementation compared to the other one?
Would it make sense to mention its specific strengths within its CORE-16145-comment maybe?

Reply:
Unless there is really some benefit i here that I would miss, I would recommend to simply strip that second implementation entirely.

Reply:
> I don't think the taskmgr ever should use anything from kernel32_vista. Neither today, nor in the future. Other applications, like Mozilla Firefox can absolutely offer similar functionality, without using that. What is the benefit of this GetProcessExecutablePath() implementation compared to the other one? Would it make sense to mention its specific strengths within its CORE-16145-comment maybe?

It's just easier to use and don't requires path conversion.

Reply:
> Unless there is really some benefit in here that I would miss, I would recommend to simply strip that second implementation entirely.

OK, I'll remove.